rust/htp/README.md

```diff
@@ -0,0 +1,67 @@
+# LibHTP
```

Comment:
Two suggestions below, as per https://github.com/OISF/suricata/pull/12841#discussion_r2020994916.

I have another suggestion that keeps some of the goals, not sure what is preferred here, so went for now with something that aligns with the one recent merged to the LibHTP repo.

Reply:
thanks